streaming/index.js

```diff
@@ -451,6 +455,7 @@ const startServer = async () => {
 
         if (result.rows.length === 0) {
           err = new Error('Invalid access token');
+          // @ts-ignore
           err.status = 401;
```

Comment:
This is being refactored separately, as I've a branch that introduces proper Error objects for the various errors we throw in the code.